packages/charm/src/manager.ts

```diff
@@ -1098,6 +1099,104 @@ export class CharmManager {
     await this.runtime.idle();
     await this.synced();
   }
+
+  /**
+   * Get the favorites cell from the home space (singleton across all spaces)
+   * @private
+   */
+  private getHomeFavorites(): Cell<Cell<unknown>[]> {
+    // Use userIdentityDID which is the user's actual identity, not the space identity
+    const homeSpace = this.runtime.userIdentityDID;
+    if (!homeSpace) {
+      throw new Error(
+        "User identity DID not available - cannot access favorites",
+      );
+    }
+    const homeSpaceCell = this.runtime.getCell(
+      homeSpace,
+      homeSpace,
+      spaceCellSchema,
+    );
+    return homeSpaceCell.key("favorites").asSchema(charmListSchema);
+  }
+
+  /**
+   * Add a charm to the user's favorites (in home space)
+   * @param charm - The charm to add to favorites
+   */
+  async addFavorite(charm: Cell<unknown>): Promise<void> {
+    const favorites = this.getHomeFavorites();
+    await favorites.sync();
+
+    const id = getEntityId(charm);
+    if (!id) return;
+
+    await this.runtime.editWithRetry((tx) => {
+      const favoritesWithTx = favorites.withTx(tx);
+      const current = favoritesWithTx.get() || [];
+
+      // Check if already favorited
+      if (current.some((c) => isSameEntity(c, id))) return;
+
+      favoritesWithTx.push(charm);
+    });
+
+    await this.runtime.idle();
+  }
+
+  /**
+   * Remove a charm from the user's favorites (in home space)
+   * @param charm - The charm or entity ID to remove from favorites
+   * @returns true if the charm was removed, false if it wasn't in favorites
+   */
+  async removeFavorite(charm: Cell<unknown> | EntityId): Promise<boolean> {
+    const id = getEntityId(charm);
+    if (!id) return false;
+
+    const favorites = this.getHomeFavorites();
+    await favorites.sync();
+
+    let removed = false;
+    await this.runtime.editWithRetry((tx) => {
+      const favoritesWithTx = favorites.withTx(tx);
+      const filtered = filterOutEntity(favoritesWithTx, id);
+      if (filtered.length !== favoritesWithTx.get().length) {
+        favoritesWithTx.set(filtered);
+        removed = true;
+      }
+    });
+
+    return removed;
+  }
+
+  /**
+   * Check if a charm is in the user's favorites (in home space)
+   * @param charm - The charm or entity ID to check
+   * @returns true if the charm is favorited, false otherwise
+   */
+  isFavorite(charm: Cell<unknown> | EntityId): boolean {
```

Comment:
Is the `| EntityId` necessary here? We should remove those eventually so if the call site doesn't need it, let's not add it. Otherwise keep it.